pkg/controller/cluster/status.go

```diff
@@ -162,6 +162,41 @@ func (c *Controller) updateCertificatesWithRetry(ctx context.Context, tenant *mi
 	return t, nil
 }
 
+func (c *Controller) updateCustomCertificatesStatus(ctx context.Context, tenant *miniov2.Tenant, customCertificates *miniov2.CustomCertificates) (*miniov2.Tenant, error) {
+	return c.updateCustomCertificatesWithRetry(ctx, tenant, customCertificates, true)
+}
+
+func (c *Controller) updateCustomCertificatesWithRetry(ctx context.Context, tenant *miniov2.Tenant, customCertificates *miniov2.CustomCertificates, retry bool) (*miniov2.Tenant, error) {
```

Comment:
Suggestion: https://pkg.go.dev/k8s.io/client-go/util/retry#RetryOnConflict

which is a wrapper function and can be used as 

```go
err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
    // STEP 1: Get the object
    // STEP 2: Edit the object
    // STEP 3: Update the object
})
if err != nil {
    return err
}
```

Reply:
Removed retry. Synch handler deals with this, as you mention.